src/set_theory/game/basic.lean

```diff
@@ -48,71 +43,45 @@ open pgame
-by { rintro ⟨x⟩ ⟨y⟩, change _ → _ ∨ ⟦x⟧ = ⟦y⟧, rw quotient.eq, exact lt_or_equiv_of_le }
-
-instance : is_trichotomous game (<) :=
-⟨by { rintro ⟨x⟩ ⟨y⟩, change _ ∨ ⟦x⟧ = ⟦y⟧ ∨ _, rw quotient.eq, apply lt_or_equiv_or_gt }⟩
-
-@[simp] theorem not_le : ∀ {x y : game}, ¬ x ≤ y ↔ y < x :=
-by { rintro ⟨x⟩ ⟨y⟩, exact not_le }
-
-@[simp] theorem not_lt : ∀ {x y : game}, ¬ x < y ↔ y ≤ x :=
-by { rintro ⟨x⟩ ⟨y⟩, exact not_lt }
+instance : add_comm_group game :=
+{ zero := ⟦0⟧,
+  neg := quot.lift (λ x, ⟦-x⟧) (λ x y h, quot.sound (@neg_congr x y h)),
+  add := quotient.lift₂ (λ x y : pgame, ⟦x + y⟧)
+    (λ x₁ y₁ x₂ y₂ hx hy, quot.sound (pgame.add_congr hx hy)),
+  add_zero := by { rintro ⟨x⟩, exact quot.sound (add_zero_equiv x) },
+  zero_add := by { rintro ⟨x⟩, exact quot.sound (zero_add_equiv x) },
+  add_assoc := by { rintros ⟨x⟩ ⟨y⟩ ⟨z⟩, exact quot.sound add_assoc_equiv },
+  add_left_neg := by { rintro ⟨x⟩, exact quot.sound (add_left_neg_equiv x) },
+  add_comm := by { rintros ⟨x⟩ ⟨y⟩, exact quot.sound add_comm_equiv } }
 
-instance : has_zero game := ⟨⟦0⟧⟩
-instance : inhabited game := ⟨0⟩
 instance : has_one game := ⟨⟦1⟧⟩
+instance : inhabited game := ⟨0⟩
 
-/-- The negation of `{L | R}` is `{-R | -L}`. -/
-instance : has_neg game :=
-⟨quot.lift (λ x, ⟦-x⟧) (λ x y h, quot.sound (@neg_congr x y h))⟩
+instance : partial_order game :=
+{ le := quotient.lift₂ (λ x y, x ≤ y) (λ x₁ y₁ x₂ y₂ hx hy, propext (le_congr hx hy)),
+  le_refl := by { rintro ⟨x⟩, exact le_refl x },
+  le_trans := by { rintro ⟨x⟩ ⟨y⟩ ⟨z⟩, exact @le_trans _ _ x y z },
+  le_antisymm := by { rintro ⟨x⟩ ⟨y⟩ h₁ h₂, apply quot.sound, exact ⟨h₁, h₂⟩ } }
 
-/-- The sum of `x = {xL | xR}` and `y = {yL | yR}` is `{xL + y, x + yL | xR + y, x + yR}`. -/
-instance : has_add game :=
-⟨quotient.lift₂ (λ x y : pgame, ⟦x + y⟧) (λ x₁ y₁ x₂ y₂ hx hy, quot.sound (pgame.add_congr hx hy))⟩
+/-- The less or fuzzy relation on games.
 
-instance : add_semigroup game.{u} :=
-{ add_assoc := by { rintros ⟨x⟩ ⟨y⟩ ⟨z⟩, exact quot.sound add_assoc_equiv },
-  ..game.has_add }
+If `0 ⧏ x` (less or fuzzy with), then Left can win `x` as the first player. -/
+def lf : game → game → Prop :=
+quotient.lift₂ lf (λ x₁ y₁ x₂ y₂ hx hy, propext (lf_congr hx hy))
 
-instance : add_monoid game :=
-{ add_zero := by { rintro ⟨x⟩, exact quot.sound (add_zero_equiv x) },
-  zero_add := by { rintro ⟨x⟩, exact quot.sound (zero_add_equiv x) },
-  ..game.has_zero,
-  ..game.add_semigroup }
+local infix ` ⧏ `:50 := lf
 
-instance : add_group game :=
-{ add_left_neg := by { rintro ⟨x⟩, exact quot.sound (add_left_neg_equiv x) },
-  ..game.has_neg,
-  ..game.add_monoid }
+@[simp] theorem not_le : ∀ {x y : game}, ¬ x ≤ y ↔ y ⧏ x :=
+by { rintro ⟨x⟩ ⟨y⟩, exact pgame.not_le }
 
-instance : add_comm_semigroup game :=
-{ add_comm := by { rintros ⟨x⟩ ⟨y⟩, exact quot.sound add_comm_equiv },
-  ..game.add_semigroup }
+@[simp] theorem not_lf : ∀ {x y : game}, ¬ x ⧏ y ↔ y ≤ x :=
+by { rintro ⟨x⟩ ⟨y⟩, exact pgame.not_lf }
```

Comment:
I'm uncertain about having both of these as simp lemmas. Perhaps it would be good to articulate the prefered simp norm form for inequalities on `pgame`?

E.g. is it "as few negations as possible", or "avoid ⧏"?

Documenting the intention (perhaps in the module doc) would be ideal.

Reply:
I think we most certainly want both of these as `simp` lemmas. The goal is not to avoid `⧏`, but make full use of it. There's a lot of useful lemmas like `move_left_lf` or `lf_def_lt` that are used throughout.

Yes, in principle, all of these lemmas could be written using negations. However, this is quite an inconvenience, since we need to define `⧏` in either case, as part of `le_lf` (even if we don't give it notation). So we'd need to prove this equivalence, and then rewrite all of the lemmas for `⧏` in terms of `¬ y ≤ x`. This would make all the lemmas about transitivity (`lf_of_le_of_lf` and so on) much harder to apply (no more potential dot notation). I imagine that this would also make rewrites and `simp` lemmas harder to use.

One last reason I prefer using `⧏` is that it makes some lemmas quite easier to remember. Just like there's a form of "antisymmetry" for `≤` (`x ≤ y` and `y ≤ x` implies `x ≈ y`), there's also a form of "antisymmetry" for `⧏` (`x ⧏ y` and `y ⧏ x` implies `x ∥ y`).

Reply:
I've added the following docstring to both lemmas:
```lean
/-- On `pgame`, simp-normal theorems should use as few negations as possible. -/
```